draft-irtf-cfrg-frost.md

```diff
@@ -284,23 +277,11 @@ little-endian integer, is a value greater than or equal to 0, and less than `Ord
 FROST requires the use of a cryptographically secure hash function, generically
 written as H, which functions effectively as a random oracle. For concrete
 recommendations on hash functions which SHOULD BE used in practice, see
-{{ciphersuites}}. By construction, the hash functions inputs MUST be smaller
-than 2^64 bits.
-
-Using H, we introduce two separate domain-separated hashes, H1, H2, and H3.
-These hash functions differ per parameter set, so H1, H2, and H3 differ
-between instantiations of the protocol as follows:
-
-~~~
-H1(m) = H(contextString || "rho" || I2OSP(len(m), 8) || m)
-H2(m) = H(contextString || "chal" || I2OSP(len(m), 8) || m)
-H3(m) = H(m)
-~~~
-
-Normally H3 would also include a domain separator, but for backwards compatibility
-with {{!RFC8032}}, it is omitted here.
-
-The variable contextString is unique for each ciphersuite defined in {{ciphersuites}}.
+{{ciphersuites}}. Using H, we introduce three separate domain-separated hashes,
+H1, H2, and H3, where H1 and H2 map arbitrary inputs to non-zero Scalar elements of
+the prime-order group scalar field, and H3 is an alias for H. The details of
```

Comment:
`H3` is an alias for `H` due to backwards compatibility reasons with {{!RFC8032}}

Reply:
What do we want H3 to be for non-RFC8032 suites?

Reply:
I picked something arbitrary for domain separation of H3 with these suites.